src/utils/actions.js

```diff
@@ -428,10 +436,17 @@ function createBundle(records, type) {
   return bundle;
 }
 
-function evaluateExpression(expression, resources, variables = {}) {
+function evaluateExpression(expression, resources, variables = {}, library, patientId) {
```

Comment:
I actually removed the function since the pieces of logic are basically one liners and performed the language check in `check-trigger-codes`. Let me know if you still think it would be better to break them up into separate functions.